pkg/mindpak/testdata/t2/manifest.json

```diff
@@ -1,16 +1,14 @@
 {
   "metadata": {
     "name": "t2",
+    "namespace": "stacklok",
     "version": "v0.0.1",
-    "date": {
-      "seconds": 1709881152,
-      "nanos": 854906417
-    }
+    "date": null
   },
   "files": {
     "profiles": [
       {
-        "name": "branch-protection.yaml",
+        "name": "branch-protection-github-profile.yaml",
```

Comment:
Changed the filename to match the profile name. I think this might be a useful constraint to enforce on bundles since it simplifes the process of mapping a profile to its path inside the bundle

Reply:
Hehe this is dancing around the same issue I noted below. Sorry, I missed it before sending my comment :)